client/packages/common/src/hooks/useQueryParams/types.ts

```diff
@@ -3,7 +3,7 @@ import { Column } from '../../ui/layout/tables';
 
 export interface FilterByConditionByType {
   string: 'equalTo' | 'like' | 'notEqualTo';
-  date: 'beforeOrEqualTo' | 'afterOrEqualTo' | 'equalTo';
+  date: 'beforeOrEqualTo' | 'afterOrEqualTo' | 'equalTo' | 'between';
```

Comment:
Added a between condition that uses beforeOrEqualTo and afterOrEqualTo from the range URL param~